### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by puneith**
@jhseu Please review.

### CODE REVIEW

File: hadoop/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ installed.
 
 2. [Apache Maven](https://maven.apache.org/)
 
-3. Tested with Hadoop 2.6.0. Patches are welcome if there are incompatibilities
+3. Downloads protos from [googleapis](https://github.com/tensorflow/tensorflow)
```
Comment (jhseu) [inline hadoop/README.md]: How about `3. Download and compile protos from [TensorFlow](https://github.com/tensorflow/tensorflow).`

Comment (puneith) [inline hadoop/README.md]: done

**Comment by jhseu** [comment]
Hmm, CLA check didn't happen automatically. Ping @googlebot

**Comment by puneith** [comment]
CLA done.

**Pull request merged**